libs/components/core/testing/src/modules/overlay/overlay-harness-filters.ts

```diff
@@ -1,8 +1,8 @@
-import { SkyHarnessFilters } from '../../shared/harness-filters';
+import { BaseHarnessFilters } from '@angular/cdk/testing';
```

Comment:
I must be missing something, what's the reason for changing the base class here?

Reply:
all that `SkyHarnessFilter` does is add `dataSkyId` as a filter. u cant add that to an overlay so theres no reason for it to extend the `SkyHarnessFilter`